docs/source/torch.rst

```diff
@@ -336,6 +336,10 @@ Reduction Ops
 
     argmax
     argmin
+    amax
+    amin
+    max
```

Comment:
This is a better place for max and min but would you please remove them from the "Comparison Ops," too, then? People should be using maximum and minimum, anyway, for comparisons.

Reply:
done